package.json

```diff
@@ -2,19 +2,20 @@
   "name": "itowns",
   "version": "0.0.0-alpha",
   "description": "A JS/WebGL framework for 3D geospatial data visualization",
-  "main": "dist/itowns.js",
+  "main": "lib/Main.js",
   "scripts": {
     "lint": "eslint \"src/**/*.js\" \"test/**/*.js\" \"examples/**/*.js\"",
     "doc": "jsdoc src/Core/Commander/Interfaces/ApiInterface/*",
     "doclint": "npm run doc -- -t templates/silent",
     "test": "npm run lint && npm run build && npm run doclint",
     "build": "webpack -p",
     "start": "webpack-dev-server -d --inline --hot",
-    "prepublish": "npm run build"
+    "prepublish": "npm run build && BABEL_DISABLE_CACHE=1 babel src --out-dir lib"
```

Comment:
So this conversation came back, as I reverted some change. I actually don't know :-) 

Maybe test should run `lint` and `doclint` only? And then prepublish could run `build`, `test` and babel? But as we run lint and tests from a PR, it's not very important for it to do so (though it gives another layer of security of course). WDYT?